blazor/chart/chart-types/radar.md

```diff
@@ -99,7 +99,7 @@ To customize the start angle of the [Radar Chart](https://help.syncfusion.com/cr
 
 ### Coefficient in axis
   
-To customize the radius of the [Radar Chart](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeriesType.html#Syncfusion_Blazor_Charts_ChartSeriesType_Radar), use [Coefficient](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartCommonAxis.html#Syncfusion_Blazor_Charts_ChartCommonAxis_Coefficient) property. By default, [Coefficient](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartCommonAxis.html#Syncfusion_Blazor_Charts_ChartCommonAxis_Coefficient) value is **100**.
+To customize the radius of the [Radar Chart](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeriesType.html#Syncfusion_Blazor_Charts_ChartSeriesType_Radar), use [Coefficient](https://help.syncfusion.com/cr/blazor#Syncfusion_Blazor_Charts_ChartCommonAxis_Coefficient) property. By default, [Coefficient](https://help.syncfusion.com/cr/blazor#Syncfusion_Blazor_Charts_ChartCommonAxis_Coefficient/Syncfusion.Blazor.html) value is **100**.
```

Comment:
Recheck the URL. If you have doubt to find the URL. 
First open corresponding platform cr link
https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.html
<img width="946" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/44014283/66b0e4ef-a190-4e9a-9dd6-c96d824a3a4a">
then add the control name in left side search box.
<img width="949" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/44014283/4acd59c8-4867-438e-ad10-2f551af4b4c8">